### ISSUE TITLE
Add a surgeon preset

### ISSUE THREAD

**Issue by HeyItsAdam** [openstreetmap/iD#7272]
Would be nice to have, could also have a tag for the surgeon's speciality (plastic surgeon, oral surgeon, etc.).

**Comment by ferdinand0101** [comment]
I would like to suggest to expand the suggestion to all the healthcare:speciality keys

**Comment by quincylvania** [comment]
Hi and welcome, @HeyItsAdam. Do you have a tag in mind for this? Healthcare tagging isn't super well defined.

**Comment by maro-21** [comment]
I don't think we should have a separate preset for each of the doctors... There are a lot of specialities
If you want to add a surgeon, you can add
```
amenity=doctors
healthcare=doctor
healthcare:speciality=surgery
```
or `healthcare:speciality=plastic_surgery` or `healthcare:speciality=oral_surgery`.

Instead of adding preset(s), I suggest to edit Doctor preset and add all the popular specialities to
`terms: medic*,physician`
so that they are easier to find.

> I would like to suggest to expand the suggestion to all the healthcare:speciality keys

Yeah. It would be good this field work similar to `cuisine` field. When I type "ser" in the cuisine field I have suggestions like "dessert/brasserie/serbian/rotisserie" but when I type "surg" in the healthcare:speciality field, I don't have such suggestions. Do they work differently?

**Comment by ferdinand0101** [comment]
so could it be implemented those presets for doctors and hospital presets ?

**Comment by ferdinand0101** [comment]
what I have done there is added a cuisine like tag suggestion box (not sure if it works) and one preset

**Comment by ferdinand0101** [comment]
note I just realized presets for hospitals aren't needed and should take care of them selves once the doctor presets exist

**Comment by bhousel** [comment]
> I don't think we should have a separate preset for each of the doctors... There are a lot of specialities.

Agreed.. sorry we didn't close this sooner!
